.travis.yml

```diff
@@ -4,11 +4,9 @@ branches:
         - master
         - qc
 node_js:
-    - "0.12.5"
-    - "4"
-    - "6"
```

Comment:
If this is a statement of dropping support for these versions than the package should be a major bump.

Reply:
So I created a PR `https://github.com/CoursePark/NestHydrationJS/pull/34` with only an empty commit on master and it fails in 0.12. So I'm not sure who it passed previously or if it did but any reasonable path forward involves dropping 0.12. It is just a question of whether this must be a major bump. As it is maintaining 0.12 support make security updates impossible.

Reply:
went with version 2